Src/Couchbase.Linq.Tests/BeerSampleTests.cs

```diff
@@ -652,5 +652,126 @@ orderby brewery.Name
                 }
             }
         }
+
+        [Test()]
+        public void AggregateTests_SimpleAverage()
+        {
+            using (var cluster = new Cluster())
```

Comment:
This should be using (var cluster = new Cluster(TestConfigurations.DefaultConfig() ){..} so that it's easier to run the test off of local host (by changing a single location in the app.config).